### PR TITLE
Add setting to display gui line

### LINKED ISSUE #28: Add option to remove gui line
Could you add option to remove gui line that is showed above minibuffer?

### PR DESCRIPTION
**Pull request by andreyorst**
fix #28

### CODE REVIEW

File: mini-modeline.el [hunk 1]
```diff
@@ -316,28 +321,33 @@ BODY will be supplied with orig-func and args."
   "Enable `mini-modeline'."
   ;; Hide modeline for terminal, or use empty modeline for GUI.
   (setq-default mini-modeline--orig-mode-line mode-line-format)
-  (setq-default mode-line-format (when (display-graphic-p) '(" ")))
+  (setq-default mode-line-format (when (and mini-modeline-display-gui-line
+                                            (display-graphic-p)) '(" ")))
```
Comment (kiennq) [inline mini-modeline.el:325]: Can you put the `'(" ")` in new line to separate from conditional phrase?

File: mini-modeline.el [hunk 2]
```diff
@@ -316,28 +321,33 @@ BODY will be supplied with orig-func and args."
...
   ;; Do the same thing with opening buffers.
   (mapc
    (lambda (buf)
      (with-current-buffer buf
        (when (local-variable-p 'mode-line-format)
          (setq mini-modeline--orig-mode-line mode-line-format)
-         (setq mode-line-format (when (display-graphic-p) '(" "))))
+         (setq mode-line-format (when (and mini-modeline-display-gui-line
+                                           (display-graphic-p)) '(" "))))
```
Comment (kiennq) [inline mini-modeline.el:333]: Same here

**Comment by kiennq** [comment]
Thanks for your contribution, I added a few nit that's nice to have fixed.

File: mini-modeline.el (hunk 1 above)
Comment (andreyorst) [reply mini-modeline.el]: done

File: mini-modeline.el (hunk 2 above)
Comment (andreyorst) [reply mini-modeline.el]: and done)

**Pull request merged**